src/test/regress/sql/multi_name_nested.sql

```diff
@@ -0,0 +1,20 @@
+create table namenest1 (id integer primary key, user_id integer);
```

Comment:
1. We usually put a comment explaining what category of features/issues each file is testing. Look at other `.sql` files to see what style we use. Without that, it is really difficult to figure out what this file tests.
2. Why are you not including this in `multi_subquery.sql`?

Reply:
1. Updated file
2. Bug is a flaw in name resolution which subqueries were able to expose. If this should be in `multi_subquery.sql` let me know